### DISCUSSION TITLE
How to make the switch time only affect the day of the product switch

### DISCUSSION THREAD

**Discussion by hpyj** [frePPLe/frepple#545] [Q&A]
I am trying to set up the product switch time(resource setup matrices) , but found that the switch time will affect every day of scheduling, rather than only affect the day of the product switch.
Do you know how to make the switch time only affect the day of the  product switch.

thanks for any thoughts!

**Comment by jdetaeye** [comment]
Couple of notes:
- In the community edition, the setup matrix feature is NOT production-ready.  It uses algorithms that are only available in the Enterprise and Cloud Edition.
- See https://frepple.org/docs/current/user-guide/examples/resource/resource-setup-matrices.php for an example model, and https://demo.frepple.com/resource-setup-matrices/planningboard/ to see it working in the Cloud Edition
